internal/broker/bind_delete.go

```diff
@@ -35,6 +37,12 @@ func (b *UnbindEndpoint) Unbind(ctx context.Context, instanceID, bindingID strin
 	instance, err := b.instancesStorage.GetByID(instanceID)
```

Comment:
Can instance not be present yet?

Reply:
in my opinion it can happen, when the unbind is called before the provisioning is finished. The instance record in the DB is inserted even before provisioning steps are executed.

Reply:
I do not get this remark, we need discussion. Is there anything that you @ralikio suggest to change?

Reply:
Discussed in direct communication.